common/persistence/dataInterfaces.go

```diff
@@ -463,22 +389,13 @@ type (
 		RequestCancelInfos  map[int64]*persistenceblobs.RequestCancelInfo
 		SignalInfos         map[int64]*persistenceblobs.SignalInfo
 		SignalRequestedIDs  map[string]struct{}
-		ExecutionInfo       *WorkflowExecutionInfo
-		ExecutionStats      *persistenceblobs.ExecutionStats
+		ExecutionInfo       *persistenceblobs.WorkflowExecutionInfo
+		ExecutionState      *persistenceblobs.WorkflowExecutionState
+		NextEventID         int64
```

Comment:
This is core change: replace `WorkflowExecutionInfo` with `persistenceblobs.WorkflowExecutionInfo`. `NextEventID` and `ExecutionState` are not part of proto object (stored as separate columns) and `ExecutionStats` and `VersionHistories` are. Therefore, I am adding the former and removing the later.
Next step is to convert entire `WorkflowMutableState` to proto.